presto-main/src/test/java/com/facebook/presto/sql/planner/optimizations/TestReorderWindows.java

```diff
@@ -246,9 +246,8 @@ public void testNotReorderAcrossNonPartitionFilter()
                         window(windowMatcherBuilder -> windowMatcherBuilder
                                         .specification(windowA)
                                         .addFunction(functionCall("avg", commonFrame, ImmutableList.of(QUANTITY_ALIAS))),
-                                filter(
-                                        RECEIPTDATE_ALIAS + " IS NOT NULL",
-                                        project(
+                                project(
+                                        filter(RECEIPTDATE_ALIAS + " IS NOT NULL",
```

Comment:
move `RECEIPTDATE_ALIAS + " IS NOT NULL"` to new line

Reply:
The rest of the file typically has the arguments related to that node on the same line with source nodes placed on a new line. That's the convention I followed here, but I'll post an update with your suggestions. Let me know what you think is better.